react/components/Input/index.js

```diff
@@ -154,7 +154,6 @@ class Input extends Component {
     switch (size) {
       case 'small':
         prefixSuffixGroupClasses += 'h-small '
-        classes += `${!token ? 't-small' : ''} `
```

Comment:
So is `t-body` the default here? I see other sizes doing this 👇 
```js
classes += `${!token ? 't-body' : ''}`
```

Reply:
Yup, it defaults to `t-body`. Do you think it would be good to add it just for safety?

Reply:
It can be a good thing to avoid confusion about other sizes when actually they'll be the same :)